app/views/group_assignments/_group_assignment_form_options.html.erb

```diff
@@ -4,6 +4,12 @@
 </dl>
 
 <dl class="form">
+  <dt><%= f.label :slug %></dt>
+  <dd><%= f.text_field :slug, class: 'textfield' %></dd>
+  <span class="note">Slug is the user friendly and URL valid name of an assignment. It will also be used as the name prefix of assignment repositories.</span>
```

Comment:
What about instead of `Slug` we just say `Assignment repository prefix` with the same explanation?